src/gdcdictionary/schemas/demographic.yaml

```diff
@@ -141,6 +141,940 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  country_of_birth:
```

Comment:
This property seems like it's going to have issues with history and time. For example, there are people who were born in countries that no longer exist, such as Yugoslavia or the USSR. Using only countries that exist today suggests that we're going to migrate this property over time as countries come in and out of existence.  

Anyway, aside from which enums to include, the real problem here probably belongs to groups like IMS, no us.